web3js-ext/src/web3/account.ts

```diff
@@ -47,4 +75,103 @@ export const signTransactionAsFeePayer = async (
 		rawTransaction: rawTx,
 		transactionHash: bytesToHex(txHash),
 	};
+};
+
+/**
+ * Get an Account object from the privateKey with context
+ * 
+ * @param context - web3 context
+ * @param privateKey - String or Uint8Array of 32 bytes
+ * @param ignoreLength - if true, will not error check length
+ * @returns A Web3Account object
+ *
+ * ```ts
+ * let provider = new Web3.providers.HttpProvider("https://public-en-baobab.klaytn.net");
+ * let web3 = new KlaytnWeb3(provider);
+ * web3.eth.accounts.privateKeyToAccount("0x348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709");
+ * >    {
+ * 			address: '0xb8CE9ab6943e0eCED004cDe8e3bBed6568B2Fa01',
+ * 			privateKey: '0x348ce564d427a3311b6536bbcff9390d69395b06ed6c486954e971d960fe8709',
+ * 			sign,
+ * 			signTransaction,
+ * 			encrypt,
+ * 	}
+ * ```
+ */
+export const privateKeyToAccountWithContext = (context: Web3Context, privateKey: Bytes, ignoreLength?: boolean): Web3Account => {
+	const privateKeyUint8Array = parseAndValidatePrivateKey(privateKey, ignoreLength);
+
+	return {
+		address: privateKeyToAddress(privateKeyUint8Array),
+		privateKey: bytesToHex(privateKeyUint8Array),
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		// @ts-ignore
+		signTransaction: async (otx: Transaction): Promise<SignTransactionResult> => {
+			// @ts-ignore
+			let tx = await prepareTransaction(otx, context, privateKey);
+			let priv = bytesToHex(privateKey);
+			return signTransaction(tx, priv);
+		},
+		sign: (data: Record<string, unknown> | string) =>
+			sign(typeof data === 'string' ? data : JSON.stringify(data), privateKeyUint8Array),
+		encrypt: async (password: string, options?: Record<string, unknown>) =>
+			encrypt(privateKeyUint8Array, password, options),
+	};
```

Comment:
```suggestion
  // web3/src/accounts.ts:initAccountsForContext
  const account = privateKeyToAccount(privateKey);
  return {
    ...account,
    signTransaction: async (transaction: Transaction) =>
      signTransactionWithContext(transaction, account.privateKey),
  };
```

Reply:
Wow, I admire your insight about the code.
Thank you very much.